src/marqo/s2_inference/languagebind/__init__.py

```diff
@@ -0,0 +1,74 @@
+import torch
+from torch import nn
+from transformers import AutoConfig
+
+from .image.configuration_image import LanguageBindImageConfig
+from .image.modeling_image import LanguageBindImage
+from .image.tokenization_image import LanguageBindImageTokenizer
+from .image.processing_image import LanguageBindImageProcessor
+
+from .video.configuration_video import LanguageBindVideoConfig
+from .video.modeling_video import LanguageBindVideo
+from .video.tokenization_video import LanguageBindVideoTokenizer
+from .video.processing_video import LanguageBindVideoProcessor
+
+from .audio.configuration_audio import LanguageBindAudioConfig
+from .audio.modeling_audio import LanguageBindAudio
+from .audio.tokenization_audio import LanguageBindAudioTokenizer
+from .audio.processing_audio import LanguageBindAudioProcessor
+
+
+config_dict = {
+    'image': LanguageBindImageConfig,
+    'video': LanguageBindVideoConfig,
+    'audio': LanguageBindAudioConfig
+}
+model_dict = {
+    'image': LanguageBindImage,
+    'video': LanguageBindVideo,
+    'audio': LanguageBindAudio
+}
+transform_dict = {
+    'video': LanguageBindVideoProcessor,
+    'audio': LanguageBindAudioProcessor,
+    'image': LanguageBindImageProcessor,
+}
+
+class LanguageBind(nn.Module):
```

Comment:
Is this test code? Otherwise init file isn't the right place

Reply:
The whole `/languagebind/` folder was copied from their repo and so I didn't modify anything there. Perhaps we can optimize it a bit.

Reply:
Update, I modified some things in their code so we can reduce our dependencies